src/API/Trace/SpanContext.php

```diff
@@ -161,4 +85,26 @@ public function getTraceFlags(): int
     {
         return $this->traceFlags;
     }
+
+    /** @inheritDoc */
+    public static function createSpanContext(string $traceId, string $spanId, bool $isRemote, int $traceFlags = self::TRACE_FLAG_DEFAULT, ?TraceStateInterface $traceState = null): SpanContextInterface
+    {
+        return new SpanContext(
+            $traceId,
+            $spanId,
+            $traceFlags,
+            $isRemote,
+            $traceState,
+        );
+    }
+
+    /** @inheritDoc */
+    public static function getInvalid(): SpanContextInterface
+    {
+        if (null === self::$invalidContext) {
+            self::$invalidContext = self::createSpanContext(self::INVALID_TRACE, self::INVALID_SPAN, false, 0);
```

Comment:
We should use public constants from SpanContextValidator rather than redeclaring them here.

Reply:
I have removed public constants from SpanContext Now